agent/src/main/java/com/spotify/ffwd/statistics/SemanticCoreStatistics.java

```diff
@@ -18,17 +18,38 @@
 import com.codahale.metrics.Counter;
 import com.codahale.metrics.Histogram;
 import com.codahale.metrics.Meter;
+import com.codahale.metrics.Metric;
+import com.codahale.metrics.SlidingTimeWindowReservoir;
 import com.spotify.metrics.core.MetricId;
+import com.spotify.metrics.core.SemanticMetricBuilder;
 import com.spotify.metrics.core.SemanticMetricRegistry;
 import eu.toolchain.async.FutureFinished;
 import java.util.concurrent.TimeUnit;
 import lombok.RequiredArgsConstructor;
 
 @RequiredArgsConstructor
 public class SemanticCoreStatistics implements CoreStatistics {
+    private static final int HISTOGRAM_TTL_MINUTES = 2;
     private final SemanticMetricRegistry registry;
     private final MetricId metric = MetricId.build();
 
+    /* A Histogram with a Reservoir that remembers values for a maximum of x minutes, so that slow
+     * rate doesn't cause long-lingering values in the Histogram. */
+    private static final SemanticMetricBuilder<Histogram> HISTOGRAM_BUILDER =
```

Comment:
What is the reason for instantiating an anonymous class over having a named class?

Reply:
Thanks for your review! 
My reasoning is that it's a very simple class. Ideally, it would have been possible to specify a reservoir to the default registry.histogram() method, but it's not. This is the most simple workaround to that. The upside of having it in the same file as the other related things is in my opinion greater than any added clarity that a separate file would give.

Reply:
I realise I didn't actually answer your question :) Yes, indeed, it could be a named class in the same file. In this case, tho, It would in my opinion just add lines and not clarity. Do you disagree?